source/source-connector/configuration-properties/startup.txt

```diff
@@ -0,0 +1,164 @@
+           ``1970-01-01T00:00:30Z``)
+       | * A BSON Timestamp in the canonical extended JSON (v2) format
+           (for example, ``{"$timestamp": {"t": 30, "i": 0}}``)
+
+   * - | **startup.mode.copy.existing.namespace.regex**
+     - | **Type:** string
+       |
+       | **Description:**
+       | Regular expression the connector uses to match namespaces from
+         which to copy data. A namespace describes the MongoDB database name
+         and collection separated by a period, e.g.
+         ``databaseName.collectionName``.
+
+       .. example::
+
+          In the following example, the regular expression setting matches
+          collections that start with "page" in the "stats" database.
+
+          .. code-block:: none
+
+             startup.mode.copy.existing.namespace.regex=stats\.page.*
+
+          The "\\" character in the example above escapes the "." character
+          that follows it in the regular expression. For more information on
+          how to build regular expressions, see the Java API documentation on
+          `Patterns <https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html>`__.
+
+       | **Default**: ``""``
+       | **Accepted Values**: A valid regular expression
+
+   * - | **startup.mode.copy.existing.pipeline**
+     - | **Type:** string
+       |
+       | **Description:**
+       | An inline array of :manual:`pipeline operations </core/aggregation-pipeline/>`
+         the connector runs when copying existing data. You can use this
+         setting to filter the source collection and improve the use of
+         indexes in the copying process.
+
+       .. example::
+
+          The following example shows how you can use the :manual:`$match </reference/operator/aggregation/match/>`
+          aggregation operator to instruct the connector to copy only
+          documents that contain a ``closed`` field with a value of ``false``.
+
+          .. code-block:: none
+
+             startup.mode.copy.existing.pipeline=[ { "$match": { "closed": "false" } } ]
+
+       | **Default**: ``""``
+       | **Accepted Values**: Valid aggregation pipeline stages
+
+   * - | **startup.mode.copy.existing.max.threads**
+     - | **Type:** int
+       |
+       | **Description:**
+       | The maximum number of threads the connector can use to copy data.
+       | **Default**: number of processors available in the environment
+       | **Accepted Values**: An integer
+
+   * - | **startup.mode.copy.existing.queue.size**
+     - | **Type:** int
+       |
+       | **Description:**
+       | The size of the queue the connector can use when copying data.
+       | **Default**: ``16000``
+       | **Accepted Values**: An integer
+
+   * - | **startup.mode.copy.existing.allow.disk.use**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | When set to ``true``, 
+         the connector sets the copy existing aggregation 
+         to use temporary disk storage. 
```

Comment:
Issue: Something about this phrasing sounds a little off.

Suggestion:

```suggestion
         the connector uses temporary disk storage for the copy existing aggregation 
```

Or:

```suggestion
         the connector instructs the copy existing aggregation 
         to use temporary disk storage. 
```

...or something similar.